docs/testing-deployment-publishing/branching-collaboration/branching.md

```diff
@@ -0,0 +1,233 @@
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+
+For certain conflicts, such as those involving variables and API configurations, you'll see a **View Configuration**     option. Enabling this option allows you to easily identify what has changed.
+
+![view-configuration](../imgs/view-configuration.avif)
+
+### Resolve conflicts manually
+If you choose to resolve manually, you can directly make changes in the **Accepted Changes** section. Note that if you cancel a manual resolution, you can choose to either keep or discard the changes you have made since starting the manual resolution.
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'
+}}>
+    <iframe 
+        src="https://demo.arcade.software/xZsRHG7d5rfNf5z937hZ?embed&show_copy_link=true"
+        title="Sharing a Project with a User"
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+
+:::info
+
+If the visual editor doesn't display something in the split screen UI that you need to manage during a merge, click the **Edit Project** button and edit the project as usual.
+
+:::
+
+## Merge vs Rebase
+
+The **Merge** option combines changes from different branches with a new commit, keeping a branched history. It's easier and more suitable for teams with simultaneous updates.
+
+The **Rebase** option applies branch commits in a linear history onto the main branch, which is more complex but creates a cleaner timeline. Ideal for individual work and active main branches.
+
+Let's understand the difference in detail with a scenario; Imagine you are working on a feature in your own branch while your colleague is making updates to the main branch.
+
+### Before merging or rebasing
+
+Your feature branch has two commits: Commit 1 and Commit 3 (which are your changes), and Commit 2 (made by a colleague in the main branch).
+
+![before-merge-rebase](../imgs/before-merge-rebase.avif)
+
+### After merging
+
+It creates a new merge commit. This merge commit ties together the histories of the feature and main branches. The result is a divergent history that converges with the merge commit, preserving the sequence of commits as they were originally made.
+
+![after-merging](../imgs/after-merging.avif)
```

Comment:
This image is too blurry, can you please check?

Reply:
Updated!